app/views/phase_two/options/2/1_1.html.erb

```diff
@@ -1,4 +1,4 @@
-<h2>Math Quiz  1: Contains X Errors</h2>
+<h2>Math Quiz  1: Contains X Errors 1_1</h2>
```

Comment:
Did we intend to change this?

Reply:
Had that in there to ensure I was getting the correct view files to render.  Oversight on my part in not removing it - good catch!